main/test/ca/uwaterloo/flix/language/phase/TestTyper.scala

```diff
@@ -221,12 +221,6 @@ class TestTyper extends FunSuite with TestUtils {
     expectError[TypeError.EffectGeneralizationError](result)
   }
 
-  test("TestLeq.Wildcard.04") {
-    val input = "def foo(a: Int32): Int32 & _ = a"
-    val result = compile(input, Options.TestWithLibNix)
-    expectError[TypeError.EffectGeneralizationError](result)
-  }
-
   test("TestLeq.Wildcard.05") {
     val input = "def foo(g: Int32 -> Int32 & _): Int32 & _ = g(1)"
```

Comment:
How about some tests of that use the rigidity env?

Reply:
I added some to Unification. Anything checking for a generalization error is already making use of the renv implicitly via the schemes.